jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/DefaultCredentialRetrievers.java

```diff
@@ -112,10 +115,16 @@ public DefaultCredentialRetrievers setCredentialHelperSuffix(
     if (knownCredentialRetriever != null) {
       credentialRetrievers.add(knownCredentialRetriever);
     }
-    if (credentialHelperSuffix != null) {
-      credentialRetrievers.add(
-          credentialRetrieverFactory.dockerCredentialHelper(
-              DockerCredentialHelper.CREDENTIAL_HELPER_PREFIX + credentialHelperSuffix));
+    if (credentialHelper != null) {
+      if (credentialHelper.contains(FileSystems.getDefault().getSeparator())
+          && Files.exists(Paths.get(credentialHelper))) {
```

Comment:
I think we should throw an error if the credential helper path doesn't exist or else authentication may fail invisibly.

Reply:
Can't remember exactly, but isn't our strategy trying the next credential if the current one fails? I wonder if we just print a warning and ignore the non-existing helper path (just skip adding it the return list) and proceed. It's basically a question whether we want to crash right away if the file doesn't exist. (Currently, I think if a credential helper doesn't exist, it doesn't crash?)

Reply:
Yea, we should definitely try other strategies if certain strategies like inferring the credential helper fails. However, for the case where a path is specified and the file doesn't exist, this should be an invalid input error for the user since the user defined it themselves.